pytorch_lightning/utilities/apply_func.py

```diff
@@ -161,14 +160,11 @@ def batch_to(data):
     return apply_to_collection(batch, dtype=dtype, function=batch_to)
 
 
-def convert_to_tensors(data, device: torch.device = None):
-    if device is None:
-        raise MisconfigurationException("device (torch.device) should be provided.")
```

Comment:
why is this removed?

Reply:
It seemed to me it should work fine with `device=None`. But maybe I was a bit overactive and setting `device` args as non-optional with a default value `None` and raising this message when `device is None` is more appropriate. I can change that back.